components/tor/tor_utils.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/tor/tor_utils.h"
+
+#include "base/files/file_path.h"
+#include "brave/components/tor/tor_constants.h"
+#include "chrome/common/chrome_constants.h"
+#include "chrome/common/pref_names.h"
```

Comment:
@mariospr `npm run gn_check Release` complains:
<pre>
ERROR in C:\bb3\brave-browser\src\brave\components\tor\tor_utils.cc
  Illegal include: "chrome/common/chrome_constants.h"
    Because of "-chrome" from brave's include_rules.
  Illegal include: "chrome/common/pref_names.h"
    Because of "-chrome" from brave's include_rules.
</pre>

Reply:
Also:
<pre>
Running cpplint...
components/tor/tor_utils.cc:23:  Add #include <string> for string  [build/include_what_you_use] [4]
</pre>

Reply:
Oops! Fixed both issues in https://github.com/brave/brave-core/commit/6f7bd0024d7244aa1217d94d6f432a5b5c31689b, thanks for the comment.